src/components/Spawn/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { GetSpawnTaskQuery } from "gql/generated/types";
+
+interface GetNoExpirationCheckboxTooltipCopyProps {
+  disableExpirationCheckbox: boolean;
+  isVolume: boolean;
+  limit: number;
+}
+export const getNoExpirationCheckboxTooltipCopy = ({
+  disableExpirationCheckbox,
+  isVolume,
+  limit,
+}: GetNoExpirationCheckboxTooltipCopyProps) =>
+  disableExpirationCheckbox
+    ? `You have reached the max number of unexpirable ${
+        isVolume ? `volumes` : `hosts`
+      }  (${limit}). Toggle an existing ${
+        isVolume ? "volume" : "host"
+      } to expirable to enable this checkbox.`
+    : undefined;
+
+export const validateTask = (taskData: GetSpawnTaskQuery["task"]) => {
```

Comment:
nit. Should this live in the spawnHost/utils since i don't think this is used by volumes